GREEDY/App.config

```diff
@@ -8,7 +8,7 @@
         <add key="TessDataPath" value="../../tessdata" />
         <add key="FilterImageFiles" value="Image Files(*.PNG; *.JPG; *.GIF)| *.PNG; *.JPG; *.GIF" />
         <add key="SaveFilePath" value="C:\" />
-        <add key="CategoriesDataPath" value="../../Resources/categories.txt" />
+        <add key="CategoriesDataPath" value="../../Categories.json" />
     </appSettings>
```

Comment:
Still need to think how to avoid to have a path to local repository. I recommend you to add this file to the program an read it from there. Not from local repository

Reply:
I think we should concentrate more on functionality and worry about data saving once we have a database